test/mockSchema.ts

```diff
@@ -226,7 +226,7 @@ export const mockSchema: (ERC725JSONSchema & {
     // testing data
     // the full array of values
     returnRawData: [
-      abiCoder.encodeParameter('bytes', abiCoder.encodeParameter('uint256', 2)), // array length
+      abiCoder.encodeParameter('bytes', abiCoder.encodeParameter('uint128', 2)), // array length
```

Comment:
Here you have kept the abiCode.encodeParemeter, nice. Why didn't you update it the same way earlier? cf. the other comment:

https://github.com/ERC725Alliance/erc725.js/pull/287/files#diff-5927840c3323c66acd20762ba79459d6e5fe967c32d6d2a4be8751fb589c0c93R330

Reply:
I am actually not sure about this for the mockSchemas as it is not clear to me.

Here the same reason as before. Because ABI encoding a uint128 still result in a 32 bytes long value.

But we want it to be 16 bytes long.

Reply:
Ok I was misunderstanding and getting things mixed up. 
I have updated the `mockSchemas` to use `leftPad` the same way as it is done in the library for Array length